libminifi/include/core/state/UpdatePolicy.h

```diff
@@ -122,10 +120,7 @@ class UpdatePolicyBuilder {
 
 
  protected:
-  explicit UpdatePolicyBuilder(const UpdatePolicyBuilder &other)
-      : current_policy_(other.current_policy_) {
-  }
-
+  explicit UpdatePolicyBuilder(const UpdatePolicyBuilder &other) = default;
```

Comment:
Would you mind fixing this copy ctor by removing the `explicit` specifier?

Reply:
Done.